### DISCUSSION TITLE
Comparing PM8 results with FloPy : MT3DMS

### DISCUSSION THREAD

**Discussion by Fuadqr** [modflowpy/flopy#1310] [Q&A]
Hello everyone,
I'm working with a small setup of 10 layers with 0.85x0.58x0.3m in dimensions. I'm trying to automate the MT3DMS output (Breakthorug curves) for different parameters using FloPy. But first, I tried to compare the curves by setting up the exact model in PMwin (PM8) to see if I will get the same results. However, PMwin gave me a different BTC (different time to peak and higher concentration) compared to FloPy!

as you can see in the pics below:
![1](https://user-images.githubusercontent.com/93839722/144078845-a7589064-c815-445e-842e-7aa17fb0fc2b.PNG)
![1flopy](https://user-images.githubusercontent.com/93839722/144078852-7c031fa6-b359-4269-aa30-7c4384e02007.PNG)

I have used the same solvers and the same MODFLOW version for both, I thought the BTCs will be similar, as both PM and the flopy code are based on the same equations.

Can someone explain the different results?

Ps. here you can find the PM model and the FloPy code:

PMwin model: https://we.tl/t-0IhDFWt01Y
FloPy full code and output (HTML) : https://we.tl/t-oWNnYLrHQm

**Comment by geoscience-coding** [comment]
Did you check your concentration units and other model inputs are the same values?
